taglib/toolkit/tpicture.h

```diff
@@ -0,0 +1,145 @@
+/***************************************************************************
+    copyright            : (C) 2002 - 2008 by Scott Wheeler
```

Comment:
This should reflect your contact info, since these files obviously weren't written by me 13 years ago.  ;-)